Makefile

```diff
@@ -0,0 +1,16 @@
+bin/gonews:
```

Comment:
Use Makefile vars here please.
Also suppress output with @, refer to example I sent you.